dojo/pipeline.py

```diff
@@ -110,3 +112,10 @@ def update_product_access(backend, uid, user=None, social=None, *args, **kwargs)
             if product_name not in project_names:
                 product = Product.objects.get(name=product_name)
                 Product_Member.objects.filter(product=product, user=user).delete()
+
+
+def create_user(strategy, details, backend, user=None, *args, **kwargs):
+    if not settings.SOCIAL_AUTH_CREATE_USER:
+        return
```

Comment:
I assume you tested this, but should it officially be `return None` or `return {}` instead of returning "nothing"?

Reply:
@valentijnscholten  I was asking my self  the same, but looks like it is not expecting anything to be returned back even in original code...

https://github.com/python-social-auth/social-core/blob/master/social_core/pipeline/user.py#L73
https://github.com/python-social-auth/social-core/blob/master/social_core/pipeline/user.py#L85